docs/source/en/kv_cache.md

```diff
@@ -20,7 +20,7 @@ Efficient caching is crucial for optimizing the performance of models in various
 including text generation, translation, summarization and other transformer-based applications.
 Effective caching helps reduce computation time and improve response rates, especially in real-time or resource-intensive applications.
 
-Transformers support various caching methods, leveraging [`~Cache`] classes to abstract and manage the caching logic.
+Transformers support various caching methods, leveraging "Cache" classes to abstract and manage the caching logic.
```

Comment:
Are links like [\`~Cache\`] not working? 🤔 All public classes should be linkable through [\`~ClassName\`] 

(sorry to be persistent here, I'm spending more energy on these repo-wide automatic tools :) )

Reply:
No, it didn't work for me, no idea why. Should I try again?

Reply:
Yes please :) I'll check the CI job if it doesn't work 🙏

Reply:
Sure thing! I will push the change soon and tag you when it fails :)

Reply:
It works now magically, hehe. I think there was an error and I used another type of quote-marks/brackets 😄